src/components/GirlsHooHack/FAQ/FAQComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import { isMobile } from 'react-device-detect';
+import { Collapse } from 'react-collapse';
+import { FaFontAwesome } from "react-icons/fa";
+import { FaPlus } from "react-icons/fa";
+import { FaMinus } from "react-icons/fa";
+import {Row, Col} from 'react-bootstrap';
+
+interface Props {
+  question: string;
+  answer: string;
+}
+
+function FAQComponent(props: Props) {
```

Comment:
rename to something more descriptive -- are you referring to the question themselves? Maybe Question might be more appropriate?

Reply:
as for return type, function FAQComponent(props: Props): JSX.Element